polaris-react/src/components/Columns/Columns.tsx

```diff
@@ -16,15 +16,18 @@ type Spacing = {
   [Breakpoint in BreakpointsAlias]?: SpacingSpaceScale;
 };
 
-export interface ColumnsProps {
+export interface ColumnsProps extends PropsWithChildren {
+  /** The space between columns */
   spacing?: Spacing;
+  /** The number of columns to display
```

Comment:
I think this is the default width of the column rather than the number? Because isn't the number based on the children?